src/main/Resources/assets/Old Testament (2).svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><defs><style>.cls-1{fill:none;opacity:0;}</style></defs><title>Old Testament</title><g id="Layer_2" data-name="Layer 2"><g id="Layer_1-2" data-name="Layer 1"><rect class="cls-1" width="24" height="24"/><path d="M22,20H14.94V14.25h3.83a.45.45,0,0,0,.46-.44.45.45,0,0,0-.46-.44H14.94v-.89h3.83a.44.44,0,1,0,0-.88H14.94v-.93h3.83a.45.45,0,1,0,0-.89H14.94A6.5,6.5,0,0,0,12.71,5,6.1,6.1,0,0,1,16,4a5.9,5.9,0,0,1,6,5.78Zm-8,0H2C2,16.54,2,13.09,2,9.63A6.08,6.08,0,0,1,8.11,4,6.11,6.11,0,0,1,14,9.78ZM8,5.78a4.22,4.22,0,0,0-4.1,4v8.44h8.3c0-2.85.06-5.7,0-8.55A4.17,4.17,0,0,0,8,5.78ZM7.57,16H5.72a.45.45,0,1,1,0-.89H7.57a.45.45,0,1,1,0,.89Zm2.77-1.7H5.72a.45.45,0,0,1-.46-.44.45.45,0,0,1,.46-.44h4.62a.45.45,0,0,1,.46.44A.45.45,0,0,1,10.34,14.25Zm0-1.77H5.72a.44.44,0,1,1,0-.88h4.62a.44.44,0,1,1,0,.88Zm0-1.81H5.72a.45.45,0,1,1,0-.89h4.62a.45.45,0,1,1,0,.89Z"/></g></g></svg>
```

Comment:
any reason for the (2)? can we make the spaces "-" or "_"?

Reply:
no that's just how it was locally after download. (2) is there bc i downloaded it multiple times